app/src/main/java/org/mifos/selfserviceapp/data/FundTransfer/FundTransferTemplateResponse.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.selfserviceapp.data.FundTransfer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vishwajeet
+ * @since 23/8/16.
+ */
+
+public class FundTransferTemplateResponse {
+    private List<FundTransferTemplate> fromAccountOptions = new ArrayList<>();
+    private List<FundTransferTemplate> toAccountOptions = new ArrayList<>();
+
+    public List<FundTransferTemplate> getFromAccountOptions() {
+        return fromAccountOptions;
+    }
+
+    public void setFromAccountOptions(List<FundTransferTemplate> fromAccountOptions) {
```

Comment:
Yeah. But I thought it goes with the way I am writing code. Does it make
any difference if yes then please tell me.

On Aug 24, 2016 9:14 PM, "Satya Narayan" <notifications@github.com> wrote:

> In app/src/main/java/org/mifos/selfserviceapp/data/FundTransfer/
> FundTransferTemplateResponse.java
> <https://github.com/openMF/self-service-app/pull/17#discussion_r76082712>:
>
> > +import java.util.List;
> > +
> > +/**
> > + * @author Vishwajeet
> > + * @since 23/8/16.
> > + */
> > +
> > +public class FundTransferTemplateResponse {
> > +    private List<FundTransferTemplate> fromAccountOptions = new ArrayList<>();
> > +    private List<FundTransferTemplate> toAccountOptions = new ArrayList<>();
> > +
> > +    public List<FundTransferTemplate> getFromAccountOptions() {
> > +        return fromAccountOptions;
> > +    }
> > +
> > +    public void setFromAccountOptions(List<FundTransferTemplate> fromAccountOptions) {
>
> nit: you may not need the set methods for these. Since these are usually
> populated by retrofit, you can just have the get methods here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openMF/self-service-app/pull/17/files/d657bda5ccb13f14d37e46724ec5d2a6b347c669#r76082712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AK4JNlulzciCE3wuuQ6vruldcrkyIBE_ks5qjGbUgaJpZM4JqhyH>
> .
>